libs/blocks/src/lib/search/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { SearchChip } from '@deriv-com/components';
+import { ReactNode } from 'react';
+
+export type SearchBlockProps = {
+  showSearchresults: (e: string) => void;
+  content?: ReactNode;
+  searchPlaceholder: string;
```

Comment:
`placeholder` alone I think is enough 
```suggestion
  placeholder: string;
```